README.md

```diff
@@ -1,10 +1,71 @@
-# Template Repo
-Template for new KBase repositories
-For further info on using the KBase default repo template, 
-please see the [KBase Build & Release Guide](https://github.com/kbase/.github/tree/main/guide).
+# CDM MCP Server
 
-## To Do:
+A FastAPI-based service that enables AI assistants to interact with Delta Lake tables stored in MinIO through Spark, implementing the Model Context Protocol (MCP) for natural language data operations.
 
-- [ ] Update this README.md with info about your repository
-- [ ] Modify `Dockerfile` with needed steps (assuming repo produces a Docker image)
-- [ ] Ensure all [branch rules](https://github.com/kbase/.github/blob/develop/guide/enable-branch-rules.md) & [status checks](https://github.com/kbase/.github/blob/develop/guide/enable-branch-rules.md#require-status-checks) are enabled
+
+## Quick Start
+
+1. Clone the repository:
+   ```bash
+   git clone https://github.com/kbase/cdm-mcp-server.git
+   cd cdm-mcp-server
+   ```
+
+2. Create required directories:
+   ```bash
+   mkdir -p cdr/cdm/jupyter/cdm_shared_workspace
+   ```
+
+3. Create Docker network:
+   ```bash
+   docker network create cdm-jupyterhub-network
+   ```
```

Comment:
Why is a special network needed?

Reply:
This is necessary because otherwise user's jupyterhub instance cannot communicate with the server - same setup as the cdm-jupyterhub repo.

Reply:
Can you remind me why that is?

Reply:
yea. The jupyterhub server and spark master wasn't able to find the user's Jupyterhub instance with the default network settings (at least not working for my docker compose locally).

Reply:
Oh the master couldn't contact the driver?

Reply:
yea. Master wasn't able to resolve the driver hostname.

Reply:
:

Reply:
Hmm.. why do you not need to do the same thing with the spark standalone repo?

Reply:
I see, so this won't affect the event processor. Do you need a jupyterhub instance to test this server? Seems like you could do it without one

Reply:
I think it doesn't hurt to have it. This simplifies creating test datasets and uploading them to MinIO. Next, I plan to explore integrating the MCP server directly into the JupyterHub.